src/Umbraco.Web.UI.Client/src/packages/core/extension-registry/components/extension-slot/extension-slot.element.ts

```diff
@@ -144,6 +158,34 @@ export class UmbExtensionSlotElement extends UmbLitElement {
 		return this.renderMethod ? this.renderMethod(ext, i) : ext.component;
 	};
 
+	#addEventListenersToExtensionElement() {
+		this._permitted?.forEach((initializer) => {
```

Comment:
I think we need to remove all eventListeners and add them again, cause I think we here may end up adding the same listeners multiple times to the same components.

Reply:
Maybe there is other ways.

A more techincal complex solution would be to establish awareness on which components has which event listeners, and then only add listeners for the missing once. 😬